src/ifElse.test.js

```diff
@@ -1,11 +1,42 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('should always call condition once per function call'
+    + ` for 3 times returning correct boolean values`, () => {
+    const condition = jest.fn(() => {})
+      .mockReturnValueOnce(true)
+      .mockReturnValueOnce(false)
+      .mockReturnValueOnce(true);
 
+    [1, 2, 3].forEach(() => ifElse(condition, jest.fn(), jest.fn()));
+
+    expect(condition).toHaveBeenCalled();
+    expect(condition).toHaveBeenCalledTimes(3);
+    expect(condition.mock.calls).toHaveLength(3);
+
+    expect(condition.mock.results.map(result => result.value))
+      .toEqual([true, false, true]);
   });
 
-  // write tests here
+  it('should call first if true once with value 1'
+    + 'should call second if false twice with value 2', () => {
```

Comment:
The test description is slightly misleading. It states 'should call first if true once with value 1' and 'should call second if false twice with value 2', but it doesn't clarify that the third call to `ifElse` is not being tested for its outcome. Consider rephrasing the description to better reflect the test's purpose, such as 'should call first once and second twice based on condition values'.